Shopstagram - rozwiązanie/css/helpers/mixin/media-queries/_media-queries.scss

```diff
@@ -0,0 +1,12 @@
+
+@mixin breakpoint--tablet {
+    @media screen and (max-width: 800px) {
+        @content;
+      }
+}
+
+@mixin breakpoint--desktop {
```

Comment:
plusik za próbe, ale można to jeszcze lepiej i bardziej uniwersalnie zrobic, omowimy to na zajeciach